plugins-base/XSFeatureStatus.py

```diff
@@ -15,9 +15,23 @@
 
 if __name__ == "__main__":
     raise Exception("This script is a plugin for xsconsole and cannot run independently")
-    
+
```

Comment:
whitespace changes should belong in a separate PR

Reply:
Right, I missed it.